src/Services/LightningService.cs

```diff
@@ -740,14 +755,26 @@ public static PSBT GetCombinedPsbt(ChannelOperationRequest channelOperationReque
             OperationRequestType requestype, ILogger? _logger = null)
         {
             if (channelOperationRequest == null) throw new ArgumentNullException(nameof(channelOperationRequest));
+            
+            //If the wallet is watch only, we cannot open a channel as there is no way securely sign the PSBT with SIGHASH_NONE (the internal wallet is not there)
+            if (channelOperationRequest.Wallet != null && channelOperationRequest.Wallet.IsWatchOnly)
+            {
+                const string watchOnlyWalletsCannotOpenChannels =
+                    "Watch only wallets cannot open channels";
+
+                _logger?.LogError(watchOnlyWalletsCannotOpenChannels);
```

Comment:
maybe logging more info about the channeloperationrequest could be nice, such as an id or even the wallet id which is trying to open the channel